### PR TITLE
Added .NET Framework 4.7.1 info

### PR DESCRIPTION
**Pull request by mairaw**
Another topic missing from PR #3415 

/cc @rpetrusha, @preetikr, @vivmishra

### CODE REVIEW

File: docs/framework/install/on-windows-10.md [hunk 1]
```diff
@@ -4,22 +4,21 @@ description: Learn how to install the .NET Framework on Windows 10.
...
 You may need the .NET Framework to run an app on Windows 10. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
 
-## .NET Framework 4.7
+## .NET Framework 4.7.1
 
-The .NET Framework 4.7 is included with the [Windows 10 Creators Update](https://www.microsoft.com/software-download/windows10).
+The .NET Framework 4.7.1 is included with the [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10).
 
-If you're on an earlier version of Windows 10, you need to upgrade to [Windows 10 Creators Update](https://www.microsoft.com/software-download/windows10) to get the .NET Framework 4.7.
+If you're on an earlier version of Windows 10, you need to upgrade to [Windows 10 Fall Creators Update](https://www.microsoft.com/software-download/windows10) to get the .NET Framework 4.7.1.
```
Comment (rpetrusha) [inline docs/framework/install/on-windows-10.md]: This isn't accurate; 4.7.1 is supported on Windows 10 Anniversary Update and Windows 10 Creators Update.

**Review by rpetrusha — CHANGES_REQUESTED** [review]
@mairaw, statement about Win10 support needs correction.

**Comment by mairaw** [comment]
Thanks @rpetrusha for the review. See what you think now. It seems that the previous version was inaccurate since 4.7 was also supported on the anniversary edition, right?

**Comment by rpetrusha** [comment]
Yes. Evidently this was written based on the initial 4.7 release, which was available only on Windows 10 Creators Update and was not available on downlevel systems. Then when 4.7 was released for general availability, we forgot to revise it.

**Comment by rpetrusha** [comment]
@mairaw, LGTM.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**